Resources/config/checks/apc_fragmentation.xml

```diff
@@ -10,8 +10,8 @@
 
     <services>
         <service id="liip_monitor.check.apc_fragmentation" public="false" class="%liip_monitor.check.apc_fragmentation.class%">
-            <argument>%liip_monitor.check.apc_fragmentation.warning%</argument>
-            <argument>%liip_monitor.check.apc_fragmentation.critical%</argument>
+            <argument>__liip_monitor.check.apc_fragmentation.warning__</argument>
```

Comment:
Actually heres the commit for that:

https://github.com/Postcon/LiipMonitorBundle/commit/5af871f938934ba5a3c6798b26e1aa796801f699

When using the old parameters they might have the wrong content if that specific checker is requested for a certain group. The underscores will be replaced by the correct ones in a compiler-pass.